content/en/docs/external-add-ons/mlrun/github.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "GitHub Repository"
+description = "GitHub repository for MLRun"
+weight = 999
+manualLink = "https://github.com/mlrun/mlrun"
+icon = "fa-brands fa-github"
++++
+
+MLRun is developed in the [`mlrun/mlrun`](https://github.com/SeldonIO/seldon-core) repository.
```

Comment:
This link is incorrect

Reply:
Fixed link in https://github.com/kubeflow/website/pull/3640/commits/5be9ffa805427c5ad1645c5ad879fb1b6c6ab934